Sources/SignalRClient/DefaultHttpClient.swift

```diff
@@ -8,16 +8,21 @@
 
 import Foundation
 
-class DefaultHttpClient: HttpClientProtocol {
+class DefaultHttpClient: NSObject, HttpClientProtocol {
```

Comment:
Because now we have a separate URLSessionDelegate we no longer should have to derive from `NSObject`. This would mean we should not need `super.init()` and as a result our session can be immutable (`let`)